BFAIR/INCA/INCA_reimport.py

```diff
@@ -0,0 +1,996 @@
+                ):
+                    fragment_mass = Formula(fragment_list[2]).mass + float(
+                        fragment_list[3]
+                    )
+                    time_point = fragment_list[4]
+                else:
+                    fragment_mass = Formula(fragment_list[2]).mass + float(
+                        fragment_list[4]
+                    )
+                    time_point = fragment_list[5]
+                if expt in list(simulation_info["experiment_id"]):
+                    fittedFragments.append(
+                        {
+                            "simulation_id": simulation_id,
+                            "simulation_dateAndTime": simulation_dateAndTime,
+                            "experiment_id": expt,
+                            "sample_name_abbreviation": simulation_info[
+                                "sample_name_abbreviation"
+                            ][0],
+                            "time_point": time_point,
+                            "fragment_id": fragment_id,
+                            "fragment_mass": fragment_mass,
+                            "fit_val": flux[cnt],
+                            "fit_stdev": flux_stdev[cnt],
+                            "fit_units": units,
+                            "fit_alf": fit_alf[cnt],
+                            "fit_cor": f_par_cor[cnt],
+                            "fit_cov": f_par_cov[cnt],
+                            "free": free[cnt],
+                            "used_": True,
+                            "comment_": None,
+                        }
+                    )
+                elif expt in list(simulation_info["sample_name_abbreviation"]):
+                    fittedFragments.append(
+                        {
+                            "simulation_id": simulation_id,
+                            "simulation_dateAndTime": simulation_dateAndTime,
+                            "experiment_id": simulation_info["experiment_id"][
+                                0
+                            ],
+                            "sample_name_abbreviation": expt,
+                            "time_point": time_point,
+                            "fragment_id": fragment_id,
+                            "fragment_mass": fragment_mass,
+                            "fit_val": flux[cnt],
+                            "fit_stdev": flux_stdev[cnt],
+                            "fit_units": units,
+                            "fit_alf": fit_alf[cnt],
+                            "fit_cor": f_par_cor[cnt],
+                            "fit_cov": f_par_cov[cnt],
+                            "free": free[cnt],
+                            "used_": True,
+                            "comment_": None,
+                        }
+                    )
+            else:
+                print("type not recognized")
+        return fittedFluxes, fittedFragments
+
+        # Split to make output more managable?
```

Comment:
Not sure what you mean here.

Reply:
The indentation is off. I was thinking of splitting the summarized function since it returns a lot of values. I will remove that comment